bluesky_widgets/examples/qt_search.py

```diff
@@ -9,13 +9,27 @@
 from bluesky_widgets.qt.search import QtSearches
 from bluesky_widgets.examples.utils.generate_msgpack_data import get_catalog
 from bluesky_widgets.examples.utils.add_search_mixin import columns
+from qtpy.QtWidgets import QWidget, QPushButton, QVBoxLayout
 
-from qtpy.QtWidgets import QPushButton, QVBoxLayout, QWidget
+
+class SearchListWithButton(SearchList):
+    """
+    Specialize the bluesky-widget model for the application,
```

Comment:
Is it possible to state this as

```
<One-line summary>

<Further explanation>
```

so it will render nicely in sphinx?

Reply:
Updated